### ISSUE TITLE
format-html-message, number not correctly localized in 'en' locale

### ISSUE THREAD

**Issue by sly7-7** [ember-intl/ember-intl#456]
- [x] I am on the [latest](https://github.com/jasonmit/ember-intl/releases/latest) ember-intl version
- [x] I have searched the [issues](https://github.com/jasonmit/ember-intl/issues) of this repo and believe that this is not a duplicate

### Steps to Reproduce

see: https://github.com/ember-intl/ember-intl/compare/master...sly7-7:master

The weird thing, is that it looks like the test pass on phantomjs, but not on a browser (chrome or FF):

![capture d ecran 2017-03-09 a 17 26 39](https://cloud.githubusercontent.com/assets/1826661/23759787/0f8b4846-04ee-11e7-8417-e552526d14d6.png)
![capture d ecran 2017-03-09 a 17 29 44](https://cloud.githubusercontent.com/assets/1826661/23759786/0f8917c4-04ee-11e7-892e-e6ae8faa6e67.png)
![capture d ecran 2017-03-09 a 17 25 59](https://cloud.githubusercontent.com/assets/1826661/23759785/0f847228-04ee-11e7-9e17-616590710ec8.png)

**Comment by jasonmit** [comment]
In Chrome or Firefox what does `new Intl.NumberFormat('en').format(42000)` return for you? I cannot reproduce this locally or in Travis CI.

**Comment by sly7-7** [comment]
Hmm, it returns the correct translation `42,000`. I think the problem is due to formatHtmlMessage, not formatNumber. Also, using formatMessage correctly renders `42,000` too. For now I've found a workaround, by splitting the i18n label
```
<span class="counter">{{format-number currentMessageIndex}}</span>
{{t 'message.progress_counter'}}
<span class="counter">{{format-number totalMessageCount}}</span>
```

**Comment by sly7-7** [comment]
Also, this test was correctly running on travis, which execute on phantomjs right ? And it also pass on your chrome and FF ?

**Comment by jasonmit** [comment]
@sly7-7 intl-messageformat (the underlying compiler for format-html-message) defers to IntlNumberFormat internally which is why I was curious.  https://github.com/yahoo/intl-messageformat/blob/master/dist/intl-messageformat.js#L143

> Also, this test was correctly running on travis, which execute on phantomjs right

Correct and they passed locally for me in Chrome (did not try FF).

Do you have your browsers/OS configured to another locale other than en-us?

**Comment by sly7-7** [comment]
Yes, I guess my os (Mac OS) and my browsers are configured in Front locale

**Comment by jasonmit** [comment]
Front locale?

**Comment by sly7-7** [comment]
Sorry, phone auto-correct... I meant FR.

**Comment by sly7-7** [comment]
Back to work, maybe I could test by forcing the browser locale to `en` and see how that works

**Comment by jasonmit** [comment]
Thanks @sly7-7, it should definitely help.

**Comment by sly7-7** [comment]
@jasonmit You're welcome, please ping me if I can help debugging this

**Comment by jasonmit** [comment]
I was able to track this down, fix is going into 2.13.1 once CI passes.

**Comment by sly7-7** [comment]
Thanks a lot. I will try this tomorrow.
